cogs/tas2gif.py

```diff
@@ -0,0 +1,131 @@
+from discord.ext import commands
+from discord.ext import tasks
+import discord
+import requests
+from config import celia_home
+import asyncio, aiohttp
+import os
+import subprocess, shutil
+
+gif_processing = False
+
+class tas2gif(commands.Cog):
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command()
+    async def tas2gif(self, ctx, *args):
+
+        global gif_processing
+        if (gif_processing): 
```

Comment:
I'd rather not use this global variable at all here, instead @commands.cooldown of ~1 minute (which is the longest file you can upload) with a global bucket should work just as well while being simpler and less prone to bugs